package.json

```diff
@@ -67,4 +68,4 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0"
   }
-}
\ No newline at end of file
+}
```

Comment:
This is added when using `npm pkg set` automatically, so adding this now avoids back and forth during development